### ISSUE TITLE
thread 'main' panicked; symlink issue?

### ISSUE THREAD

**Issue by dave-theunsub** [solidiquis/erdtree#30]
The `et` command works fine - except in my home directory:

```
thread 'main' panicked at 'called `Option::unwrap()` on a `None` value', src/fs/erdtree/node.rs:271:28
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

The only difference I can see is that my home directory has links from other directories.  Like this:
```
lrwxrwxrwx@   19 dave dave 26 Feb  2022 Documents -> /foo/Documents
lrwxrwxrwx@   16 dave dave 10 Jan  2022 Downloads -> /blah/Downloads
```

Any other ideas?  Let me know if you require additional information.  Using version 1.20.

[backtrace.txt](https://github.com/solidiquis/erdtree/files/10891665/backtrace.txt)

**Comment by solidiquis** [comment]
Hey @dave-theunsub thanks for bringing this to my attention. I just merged in a fix for what I believe was the cause of the issue. I will include it in the next minor version release later today. Please let me know if the issue still persists; if so please feel free to re-open the issue!

tl;dr - I don't believe it was a symlink issue but rather there may be some files that you have whose names contain non-unicode sequences which caused my `OsStr` -> `str` conversion to panic. I opted instead to do lossy conversion.
